mail_tracking/static/src/xml/mail_tracking.xml

```diff
@@ -66,7 +66,7 @@
                 </t>
                 <span class="mail_tracking o_mail_action_tracking_status"
                       t-att-data-tracking="tracking['tracking_id']"
-                      t-attf-title="Status: #{tracking['status']}">
+                      t-att-title="_.str.sprintf(_t('Status: %s'), tracking['status'])">
```

Comment:
AFAIK that's not gonna work.

For a translation string to appear in the `.pot` file, a static code scan is done across all source files. In each file, something specific is searched:

- In `.py` files, it searches for calls to `_()`, 
- in `.js` files it searches for calls to `_t()` and `_lt()`, 
- and in `.xml` files it searches for certain translatable elements and attributes. For example, the `title` attribute is translatable, but any attribute that starts with `t-` is not translated.

So if I'm not wrong, if you try to produce the `.pot` file, you'll see that this string doesn't appear. That's because you're using the JS translation system in an XML file, and the scanner won't detect it.

The easiest way to fix this problem is to provide a `status_for_humans` key in the dict, produced directly from the Python side. Also think that this way of fixing the problem could produce (if it worked), i.e. in Spanish, something like: "Estado: unknown", when you want "Estado: descconocido". Translating in the server-side will let you fix this also.